panic: ScalarMult is not available when secp256k1 is built without cgo

### Description

Got this error when running the following command on MacOS:

```bash
polycli version  
# Polygon CLI Version v0.1.33

polycli wallet inspect --raw-entropy --root-only --language english --mnemonic "..."

# Output
panic: ScalarMult is not available when secp256k1 is built without cgo

goroutine 1 [running]:
github.com/ethereum/go-ethereum/crypto/secp256k1.(*BitCurve).ScalarMult(...)
        /home/runner/go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.26/crypto/secp256k1/scalar_mult_nocgo.go:13
github.com/ethereum/go-ethereum/crypto/secp256k1.(*BitCurve).ScalarBaseMult(...)
        /home/runner/go/pkg/mod/github.com/ethereum/go-ethereum@v1.10.26/crypto/secp256k1/curve.go:252
github.com/maticnetwork/polygon-cli/hdwallet.toUncompressedPubKey(0x759e89b2bdb788bb?)
        /home/runner/work/polygon-cli/polygon-cli/hdwallet/hdwallet.go:372 +0x32
github.com/maticnetwork/polygon-cli/hdwallet.toETHAddress(0xc000fc4400?)
        /home/runner/work/polygon-cli/polygon-cli/hdwallet/hdwallet.go:363 +0x1d
github.com/maticnetwork/polygon-cli/hdwallet.(*PolyWallet).ExportRootAddress(0xc000058ae0)
        /home/runner/work/polygon-cli/polygon-cli/hdwallet/hdwallet.go:265 +0x412
github.com/maticnetwork/polygon-cli/cmd/wallet.glob..func1(0x41c8d80?, {0xc000fca070?, 0x7?, 0x7?})
        /home/runner/work/polygon-cli/polygon-cli/cmd/wallet/wallet.go:77 +0x185
github.com/spf13/cobra.(*Command).execute(0x41c8d80, {0xc000fca000, 0x7, 0x7})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:940 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001a4f00)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/maticnetwork/polygon-cli/cmd.Execute()
        /home/runner/work/polygon-cli/polygon-cli/cmd/root.go:45 +0x25
main.main()
```

Version: `polycli_0.1.33_darwin_amd64` downloaded from the releases page.

> Building the same version locally works.

--- a/Makefile
+++ b/Makefile
@@ -47,8 +47,8 @@ cross: $(BUILD_DIR) ## Cross-compile go binaries using CGO.
 	env CC=aarch64-linux-gnu-gcc CGO_ENABLED=1 GOOS=linux GOARCH=arm64 go build -ldflags "$(STATIC_LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/linux-arm64-$(BIN_NAME) main.go
 	env                          CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -ldflags "$(STATIC_LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/linux-amd64-$(BIN_NAME) main.go
   # MAC builds - this will be functional but will still have secp issues.
-	env GOOS=darwin GOARCH=arm64 go build -ldflags "$(LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/darwin-arm64-$(BIN_NAME) main.go
-	env GOOS=darwin GOARCH=amd64 go build -ldflags "$(LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/darwin-amd64-$(BIN_NAME) main.go
+	env CGO_ENABLED=1 GOOS=darwin GOARCH=arm64 go build -ldflags "$(LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/darwin-arm64-$(BIN_NAME) main.go
+	env CGO_ENABLED=1 GOOS=darwin GOARCH=amd64 go build -ldflags "$(LD_FLAGS)" -tags netgo -o $(BUILD_DIR)/darwin-amd64-$(BIN_NAME) main.go
 
 .PHONY: simplecross
 simplecross: $(BUILD_DIR) ## Cross-compile go binaries without using CGO.

